README.md

```diff
@@ -1,8 +1,9 @@
-# Computop API Module
+# ComputopAPI Module
```

Comment:
Sure this is not ComputopApi? Please double check the name in composer.json

Reply:
Please also fix https://github.com/spryker-eco/computop-api/blob/master/composer.json#L3 as per internal guidelines for module names.

Reply:
@dereuromark I've got "ComputopApi Module" with this command:
```
console dev:module:create SprykerEco.ComputopApi
```
Do I really need to use "module" instead of "Module" despite on this?

Reply:
Yes, please see core code in Development module for the regex, it is case sensitive.

Reply:
Double check the output of the tool. You are confusing readme and composer.json

    "description": "ComputopApi module",

but

    # ComputopApi Module